examples/http3-server.rs

```diff
@@ -586,6 +607,24 @@ fn handle_writable(client: &mut Client, stream_id: u64) {
     }
 
     let resp = client.partial_responses.get_mut(&stream_id).unwrap();
+
+    if let Some(ref headers) = resp.headers {
+        match http3_conn.send_response(conn, stream_id, &headers, false) {
+            Ok(_) => (),
+
+            Err(quiche::h3::Error::StreamBlocked) => {
+                return;
+            },
+
+            Err(e) => {
+                error!("{} stream send failed {:?}", conn.trace_id(), e);
+                return;
+            },
+        }
+    }
+
+    resp.headers = None;
```

Comment:
Does this mean the handling of blocked headers writes only work for one blocked iteration, or does the `return` prevent that?

Reply:
Yeah, the return means that next time the stream is writable it will try headers again.